.gitignore

```diff
@@ -29,4 +29,4 @@ _ReSharper*/
 [Tt]est[Rr]esult*
 packages/
 .vs/config/applicationhost.config
-webapp/node_modules/**/*
+.vs/
```

Comment:
Please do not remove the ignore for the node_modules folder. You should leave both ignore lines